lib/taiko.js

```diff
@@ -3725,3 +3726,29 @@ module.exports.metadata = {
     'repl',
   ],
 };
+
+/**
+ * Add interceptor for the network call. Helps in overriding request or to mock response of a network call.
+ *
+ * @example
+ * # case 1: Remove intercept for a single  URL :
+ * await resetIntercept(requestUrl)
+ * # case 2: Reset intercept for all URL :
+ * await resetIntercept()
+ *
+ * @param {string} requestUrl request URL to intercept. Optional parameters
+ */
+module.exports.resetIntercept = requestUrl => {
+  if (requestUrl) {
+    var success = networkHandler.resetInterceptor(requestUrl);
+    if (success) {
+      descEvent.emit(
+        'success',
+        'Intercepts reset for url ' + requestUrl,
+      );
+    }
+  } else {
```

Comment:
I think `resetIntercept` should inform the user when there is no intercept available for the provided `requestUrl`